march_state_machine/src/march_state_machine/gaits/tilted_path_left_knee_bend_sm.py

```diff
@@ -0,0 +1,28 @@
+import smach
+
+from march_state_machine.state_machines.step_state_machine import StepStateMachine
+from march_state_machine.states.idle_state import IdleState
+
+
+def create():
+    sm_tilted_path_left_knee_bend = smach.StateMachine(outcomes=['succeeded', 'preempted', 'failed'])
+    sm_tilted_path_left_knee_bend .register_io_keys(['sounds'])
+    with sm_tilted_path_left_knee_bend:
+        smach.StateMachine.add('GAIT TP LEFT KNEE BEND', StepStateMachine('tilted_path_left_knee_bend',
+                               subgaits=['right_open']),
+                               transitions={'succeeded': 'STANDING TP LEFT STRAIGHT', 'failed': 'failed'})
+
+        smach.StateMachine.add('GAIT TP LEFT SINGLE STEP', StepStateMachine('tilted_path_left_single_step',
+                               subgaits=['right_open', 'left_close']),
+                               transitions={'succeeded': 'STANDING TP LEFT STRAIGHT', 'failed': 'failed'})
```

Comment:
States should from now on contain `rejected` transitions if the if it's not trivial where it should go back to once the gait is rejected. In this case it should go back to `STANDING TP LEFT STRAIGHT`. Furthermore, the `failed` transition to `failed` can be removed since SMACH can imply that already.
```suggestion
                               transitions={'succeeded': 'STANDING TP LEFT STRAIGHT', 'rejected': 'STANDING TP LEFT STRAIGHT'})
```